### ISSUE TITLE
Disable preserve Focus on auto upload / download. 

### ISSUE THREAD

**Issue by robclancy** [shanalikhan/code-settings-sync#990]
https://github.com/Microsoft/vscode/issues/31474#issuecomment-319032829

Stop focusing everytime it syncs.

**Comment by shanalikhan** [comment]
You need to enable `quiteSync` configuration if you dont want to tab to write display.

**Comment by robclancy** [comment]
No. You need to write without focus like linked. Disabling the logging
isn't a fix, it's a workaround for you stealing focus.

**Comment by shanalikhan** [comment]
The more improved scenario can be.

1. when user manually trigger upload / download it should always `preserveFocus`
2. When extension triggers this on auto in the background then it should ignore `ignoreFocus`.

I think it annoys because its happening automatically on your end and therefore stealing the focus.

**Comment by robclancy** [comment]
Yes manaully it would make sense to focus as it is directly related to an
action you did yourself. But when it is from an indirect thing you do it's
just annoying, like anything on computers that steal focus unexpectedly.
